austrakka/utils/options.py

```diff
@@ -77,6 +77,31 @@ def opt_abbrev(**attrs: t.Any):
     )
 
 
+def opt_tracking_token(**attrs: t.Any):
+    defaults = {
+        'required': True,
+        'help': 'tracking token of a job, that can be used to query its status'
+    }
```

Comment:
Avoiding "job" OR be very specific - "tracking token for a dataset upload job, ..."